website/docs/targets.md

```diff
@@ -28,7 +28,9 @@ mariner2/rpm                     Builds an rpm and src.rpm.
 mariner2/rpm/debug/buildroot     Outputs an rpm buildroot suitable for passing to rpmbuild.
 mariner2/rpm/debug/sources       Outputs all the sources specified in the spec file in the format given to rpmbuild.
 mariner2/rpm/debug/spec          Outputs the generated RPM spec file
+mariner2/worker                  Builds the base worker image responsible for building the rpm
 windowscross/container (default) Builds binaries and installs them into a Windows base image
+windowscross/worker              Builds the base worker image responsible for building the rpm
```

Comment:
Can we remove the mention of "rpm" here and in any other comments/documentation, since for windows we aren't building rpms?

Reply:
You caught my copy/paste.

Reply:
Fixed.